Dockerfile

```diff
@@ -137,33 +143,30 @@ RUN systemctl enable memcached appliance-initialize evmserverd evminit evm-watch
 ## Expose required container ports
 EXPOSE 80 443
 
-# Atomic Labels
+## Atomic Labels
 # The UNINSTALL label by DEFAULT will attempt to delete a container (rm) and image (rmi) if the container NAME is the same as the actual IMAGE
 # NAME is set via -n flag to ALL atomic commands (install,run,stop,uninstall)
-
 LABEL name="manageiq" \
-          vendor="ManageIQ" \
-          version="Master" \
-          release=${REF} \
-          architecture="x86_64" \
-          url="http://manageiq.org/" \
-          summary="ManageIQ appliance image" \
-          description="ManageIQ is a management and automation platform for virtual, private, and hybrid cloud infrastructures." \
-          INSTALL='docker run -ti --privileged \
-                    --name ${NAME}_volume \
-                    --entrypoint /usr/bin/docker_initdb \
-                    $IMAGE' \
-          RUN='docker run -di --privileged \
-                    --name ${NAME}_run \
-                    -v /etc/localtime:/etc/localtime:ro \
-                    --volumes-from ${NAME}_volume \
-                    -p 80:80 \
-                    -p 443:443 \
-                    $IMAGE' \
-          STOP='docker stop ${NAME}_run && \
-          echo "Container ${NAME}_run has been stopped"' \
-          UNINSTALL='docker rm -v ${NAME}_volume ${NAME}_run && \
-          echo "Uninstallation complete"'
+      vendor="ManageIQ" \
+      version="Master" \
+      release=${REF} \
+      architecture="x86_64" \
+      url="http://manageiq.org/" \
+      summary="ManageIQ appliance image" \
+      description="ManageIQ is a management and automation platform for virtual, private, and hybrid cloud infrastructures." \
+      INSTALL='docker run -ti --privileged \
```

Comment:
What's the thought process behind running the container with the --privileged flag?